i2c/oled_i2c/oled_i2c.c

```diff
@@ -0,0 +1,301 @@
+/**
+ * Copyright (c) 2021 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <stdio.h>
+#include "pico/stdlib.h"
+#include "pico/binary_info.h"
+#include "hardware/i2c.h"
+#include "raspberry26x32.h"
+
+ /* Example code to talk to an SSD1306-based OLED display
+
+    NOTE: Ensure the device is capable of being driven at 3.3v NOT 5v. The Pico
+    GPIO (and therefore I2C) cannot be used at 5v.
+
+    You will need to use a level shifter on the I2C lines if you want to run the
+    board at 5v.
+
+    Connections on Raspberry Pi Pico board, other boards may vary.
+
+    GPIO PICO_DEFAULT_I2C_SDA_PIN (on Pico this is 4 (pin 6)) -> SDA on display
+    board
+    GPIO PICO_DEFAULT_I2C_SCK_PIN (on Pico this is 5 (pin 7)) -> SCL on
+    display board
+    3.3v (pin 36) -> VCC on display board
+    GND (pin 38)  -> GND on display board
+ */
+
+#define OLED_RESET_PIN 28
```

Comment:
Unless I'm misunderstanding what's going on here, the wiring diagram has the RST pin on the display wired to GP15? We tend to avoid GP15 due to RP2040-E5, but in any case you're setting it to GP28 here?

Reply:
This should also be mentioned in the "Connections on Raspberry Pi Pico board" comment above.

Reply:
The reset pin never ends up being used in the code, as our board  auto-resets on startup. The reset pin is mostly useful for shortcutting to sensible defaults for various configs and as we set all of them explicitly, I'll just mention it in passing in the README but remove it from the code and Fritzing diagram.